src/operator-docs/creation/fromEvent.ts

```diff
@@ -1,6 +1,251 @@
 import { OperatorDoc } from '../operator.model';
 
 export const fromEvent: OperatorDoc = {
-  'name': 'fromEvent',
-  'operatorType': 'creation'
+  name: 'fromEvent',
+  operatorType: 'creation',
+  signature: `static fromEvent(target,
+     eventName, options, selector): Observable`,
```

Comment:
nice try :D just kidding. Could you add the types as long as #269 is in discussion

Reply:
Ha! Apparently I wasn't paying attention, yes they should be there :)